pkgs/applications/video/losslesscut-bin/build-from-windows.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, p7zip
+, useMklink ? false
+, customSymlinkCommand ? null
```

Comment:
Is that unused?

Reply:
Let's just use `ln -s` since it is available in Cygwin.

I haven't found a way to setup Nixpkgs in recent Cygwin for tetsting, though.